org.serioussam.SeriousSamClassic-VK.yaml

```diff
@@ -0,0 +1,118 @@
+          - find . -name "CMakeLists.txt" -exec sed -i 's/if(NOT LOCAL_INSTALL)/if(LOCAL_INSTALL)/g' {} +
+      # Replace /usr with /app
+          - find . -name "Engine.cpp" -exec sed -i 's/\/usr\//\/app\//g' {} +
+      # Fix /app/lib (replace "//" with "/")
+          - find . -name "Engine.cpp" -exec sed -i 's/\/app\/lib\/\//\/app\/lib\//g' {} +
+      # sys_iSysPath == 0 (force library path)
+          - find . -name "Engine.cpp" -exec sed -i 's/sys_iSysPath == 0/sys_iSysPath == 1/g' {} +
+      # _fnmModLibPath = _fnmApplicationPath (force library path)
+          - find . -name "Engine.cpp" -exec sed -i 's/_fnmModLibPath = _fnmApplicationPath/_fnmModLibPath = "\/app\/lib\/" + strGameID + "\/"/g' {} +
+      # Fix SDL2 message box
+          - find . -name "Engine.cpp" -exec sed -i 's/~\/.local\/share\/Serious-Engine\///g' {} +
+          - find . -name "Engine.cpp" -exec sed -i 's/_fnmUserDir,(const char \*) strGameID/_fnmUserDir,(const char \*) _fnmUserDir/g' {} +
+      # supresses warnings for a non-used directory
+          - mkdir -p /app/share/serioussam && mkdir -p /app/share/serioussamse
+          - mkdir -p /app/lib/serioussam && mkdir -p /app/lib/serioussamse
+
+  - name: launcher
+    buildsystem: simple
+    sources:
+      - type: file
+        path: org.serioussam.SeriousSamClassic-VK.SamTFE.desktop
+      - type: file
+        path: org.serioussam.SeriousSamClassic-VK.SamTSE.desktop
+      - type: file
+        path: org.serioussam.SeriousSamClassic-VK.metainfo.xml
+      - type: file
+        url: https://raw.githubusercontent.com/tx00100xt/SeriousSamClassic-VK/main/SamTFE/serioussam.png
+        sha256: e12fdccc0ffbbd64ad2d5459d6b1da811fbe099fe8fefc0172bbf7d835648923
+        dest-filename: org.serioussam.SeriousSamClassic-VK.png
+      - type: file
+        path: org.serioussam.SeriousSamClassic-VK.128.png
+      - type: file
+        path: org.serioussam.SeriousSamClassic-VK.64.png
+      - type: file
+        path: org.serioussam.SeriousSamClassic-VK.48.png
+      - type: script
+        commands:
+          - ls ~/.var/app/org.serioussam.SeriousSamClassic-VK/data/Serious-Engine/serioussam || zenity --error --text
+            "Game data must be added to:\n~/.var/app/org.serioussam.SeriousSamClassic-VK/serioussam/"
+            --ok-label "Close" --width=400
+          - ls /app/lib/serioussam && ls /app/lib/serioussamse && ls /app/lib 
+          - serioussam "$@"
+        dest-filename: serioussam.sh
+      - type: script
+        commands:
+          - ls ~/.var/app/org.serioussam.SeriousSamClassic-VK/data/Serious-Engine/serioussamse || zenity --error --text
+            "Game data must be added to:\n~/.var/app/org.serioussam.SeriousSamClassic-VK/serioussamse/"
+            --ok-label "Close" --width=400
+          - serioussamse "$@"
+        dest-filename: serioussamse.sh
+    build-commands:
+      - install -Dm 644 org.serioussam.SeriousSamClassic-VK.SamTFE.desktop -t /app/share/applications
```

Comment:
one of this desktop file should match the app-id. The one that matches the `command`.